selfdrive/thermald/thermald.py

```diff
@@ -104,16 +103,12 @@ def handle_fan_eon(max_cpu_temp, bat_temp, fan_speed, ignition):
     # update speed if using the low thresholds results in fan speed decrement
     fan_speed = new_speed_l
 
-  if bat_temp < _BAT_TEMP_THRESHOLD:
```

Comment:
are you sure we can remove this? we still support the EON gold

Reply:
this is fine, worst case the EON fans might spin a little faster